backends/ClimaCommsMPI/src/ClimaCommsMPI.jl

```diff
@@ -79,6 +79,22 @@ mutable struct MPISendRecvGraphContext <: ClimaComms.AbstractGraphContext
     recv_reqs::Vector{MPI.Request}
 end
 
+"""
+    MPIPSendRecvGraphContext
```

Comment:
Let's give it a slightly more descriptive name (that P is hard to see)
```suggestion
    MPIPersistentSendRecvGraphContext
```